src/ref.rs

```diff
@@ -879,6 +993,53 @@ mod tests {
 
     use super::*;
     use crate::util::testutil::*;
+    use test::{self, Bencher};
+
+    #[bench]
+    fn bench_from_bytes_sized(b: &mut Bencher) {
+        let buf = Align::<[u8; 8], AU64>::default();
+        // `buf.t` should be aligned to 8, so this should always succeed.
+        let bytes = &buf.t[..];
+        b.iter(|| test::black_box(Ref::<_, AU64>::from_bytes(test::black_box(bytes)).unwrap()));
+    }
+
+    #[bench]
+    fn bench_into_ref_sized(b: &mut Bencher) {
+        let buf = Align::<[u8; 8], AU64>::default();
+        let bytes = &buf.t[..];
+        b.iter(|| {
+            let r = Ref::<_, AU64>::from_bytes(test::black_box(bytes)).unwrap();
+            test::black_box(Ref::into_ref(r))
+        });
```

Comment:
So that only the `into_ref` part is being benchmarked:

```suggestion
        let r = Ref::<_, AU64>::from_bytes(bytes).unwrap();
        b.iter(|| {
            test::black_box(Ref::into_ref(test::black_box(r)))
        });
```